src/DropdownMenu.js

```diff
@@ -55,12 +55,14 @@ const DropdownMenu = (props, context) => {
     const position2 = right ? 'end' : 'start';
     attrs.placement = `${position1}-${position2}`;
     attrs.component = tag;
+    attrs.children = [...attrs.children];
```

Comment:
I'm not quite sure I'm following. How would this resolve #1070, exactly?

This isn't a deep copy by the way, it's a shallow copy.